clerk/instances.go

```diff
@@ -106,3 +106,18 @@ func (s *InstanceService) UpdateOrganizationSettings(params UpdateOrganizationSe
 	}
 	return &organizationSettingsResponse, nil
 }
+
+type UpdateHomeURLParams struct {
+	HomeURL string `json:"home_url"`
+}
+
+func (s *InstanceService) UpdateHomeURL(params UpdateHomeURLParams) error {
+	req, _ := s.client.NewRequest(http.MethodPost, "instance/change_domain", &params)
+
+	var response interface{}
+	_, err := s.client.Do(req, &response)
```

Comment:
🔧 The endpoint doesn't return a response so we should change the code as follow. Please make sure that still will work
```suggestion
	_, err := s.client.Do(req, nil)
```

Reply:
Replace it and still working.

Reply:
Awesome 💯